### DISCUSSION TITLE
No properties are displayed on any items from the contents table or associated contents table on any folder

### DISCUSSION THREAD

**Discussion by nextor11** [microsoft/mfcmapi#539] [General]
No properties are displayed on any items from the contents table or associated contents table on any folder:
![image](https://user-images.githubusercontent.com/41907401/171388560-5fd5652a-ec45-4814-a5be-6d321d97746f.png)

These are the only options I changed from default:
![image](https://user-images.githubusercontent.com/41907401/171388347-99331bb0-f02f-450a-8373-161bad7875dd.png)

I enabled logging (0x00000001 Generic) and I got this log:
[mfcmapi.log](https://github.com/stephenegriffin/mfcmapi/files/8814420/mfcmapi.log)

**Comment by stephenegriffin** [comment]
GetPropList or GetProps is failing. You can turn on MAPI logging to get details why
